gemrb/core/GUI/GameControl.cpp

```diff
@@ -292,120 +345,154 @@ static const int arrow_orientations[16]={
 
 //Draws arrow markers along the edge of the game window
 //WARNING:don't use reference for point, because it is altered
-void GameControl::DrawArrowMarker(const Region &screen, Point p, const Region &viewport, const Color& color)
+void GameControl::DrawArrowMarker(Point p, const Color& color)
 {
-	Video* video = core->GetVideoDriver();
+	WindowManager* wm = core->GetWindowManager();
+	auto lock = wm->DrawHUD();
```

Comment:
Codacy found an issue: [Variable 'lock' is assigned a value that is never used.](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6310305)